docs/plugins/load-common-components.js

```diff
@@ -34,5 +37,8 @@ Vue.component('DocsToggleContent', DocsToggleContent);
 Vue.component('DocsToggleButton', DocsToggleButton);
 Vue.component('DocsTable', DocsTable);
 Vue.component('DocsSubNav', DocsSubNav);
+Vue.component('KIcon', KIcon);
+Vue.component('KTextTruncator', KTextTruncator);
```

Comment:
These components are already registered globally with Vue in the `lib/KThemePlugin.js` file, so we do not need to re-register them here. Only adding `DocsKCard` should work here.